multichain-testing/scripts/fetch-starship-chain-info.ts

```diff
@@ -38,10 +38,6 @@ const backmap = {
 for (const ibcInfo of ibc.data) {
   ibcInfo.chain_1.chain_name = backmap[ibcInfo.chain_1.chain_name];
   ibcInfo.chain_2.chain_name = backmap[ibcInfo.chain_2.chain_name];
-  for (const c of ibcInfo.channels) {
-    // @ts-expect-error XXX bad typedef
-    c.tags.preferred = c.tags.perferred;
-  }
```

Comment:
I think we can get rid of `L32-L41` altogether?

Reply:
yes!